pwndbg/commands/vmmap.py

```diff
@@ -284,118 +284,135 @@ def vmmap(
         )
 
 
-if pwndbg.dbg.is_gdblib_available():
-    parser = argparse.ArgumentParser(description="Add virtual memory map page.")
-    parser.add_argument("start", help="Starting virtual address")
-    parser.add_argument("size", help="Size of the address space, in bytes")
-    parser.add_argument(
-        "flags",
-        nargs="?",
-        type=str,
-        default="",
-        help="Flags set by the ELF file, see PF_X, PF_R, PF_W",
-    )
-    parser.add_argument(
-        "offset",
-        nargs="?",
-        default=0,
-        help="Offset into the original ELF file that the data is loaded from",
-    )
+parser = argparse.ArgumentParser(description="Add virtual memory map page.")
+parser.add_argument("start", type=int, help="Starting virtual address")
+parser.add_argument("size", type=int, help="Size of the address space, in bytes")
+parser.add_argument(
+    "flags",
+    nargs="?",
+    type=str,
+    default="",
+    help="Flags set by the ELF file, see PF_X, PF_R, PF_W",
```

Comment:
How does one see `PF_X` etc? :p

We should provide an example what we expect here as valid flag

Reply:
Maybe: `Flags set by the ELF file (r - read, w - write, x - executable)` if that is valid? idk

Reply:
done